crypto/fipsmodule/sha/internal.h

```diff
@@ -377,9 +378,12 @@ OPENSSL_EXPORT int SHA3_Final(uint8_t *md, KECCAK1600_CTX *ctx);
 OPENSSL_EXPORT size_t SHA3_Absorb(uint64_t A[SHA3_ROWS][SHA3_ROWS],
                                   const uint8_t *data, size_t len, size_t r);
 
-// SHA3_Squeeze generate |out| hash value of |len| bytes.
+// SHA3_Squeeze generate |out| hash value of |len| bytes. The bool first indicates
```

Comment:
```suggestion
// SHA3_Squeeze generates |out| value of |len| bytes (per call). It can be called
// multiple times when used as eXtendable Output Function. |first| indicates weather it is 
// the first call to SHA3_Squeeze; i.e., if the current block has been already processed 
// during the last call to SHA3_Absorb.
```